### ISSUE TITLE
Support screen readers

### ISSUE THREAD

**Issue by chuckharmston** [fightforthefuture/idl-members#3]
We should present a hidden, text-only message at the top of the page for users of alternative user agents, especially screen readers. This message should be marked as a WAI-ARIA live region with the appropriate [attributes](http://juicystudio.com/article/wai-aria-live-regions.php).
